Delcam.ProductInterface.PowerMILL.Test/PMEntityTests/PMProjectTests.cs

```diff
@@ -126,27 +126,25 @@ public void ActiveWorldWorkplaneTest()
         }
 
         [Test]
-        public void CreateBlockFromBoundary()
-        {   
-            _powerMILL.Execute(string.Format("CREATE BOUNDARY \"{0}\"", "TestBoundary"));            
-            _powerMILL.Execute(string.Format("EDIT BOUNDARY \"{0}\" INSERT FILE \"{1}\"", "TestBoundary", TestFiles.CurvesFiles));
+        public void CreateBlockFromBoundaryWithLimitsTest()
+        {               
+            _powerMILL.ActiveProject.Boundaries.CreateBoundary(TestFiles.CurvesFiles);
             _powerMILL.ActiveProject.Refresh();
             PMBoundary boundary = _powerMILL.ActiveProject.Boundaries.ActiveItem;
-            _powerMILL.ActiveProject.CreateBlock(boundary, 0, 100);
-            Assert.AreEqual("100,0", _powerMILL.ExecuteEx("PRINT $BLOCK.LIMITS.ZMax"));
+            Autodesk.Geometry.BoundingBox boundingBox = _powerMILL.ActiveProject.CreateBlockFromBoundaryWithLimits(boundary, 0, 100);            
```

Comment:
One nice thing with C# is that rather than having to specify the full type in the declaration you can use "var" instead.  So write:
var boundingBox = _powerMILL.ActiveProject.CreateBlockFromBoundaryWithLimits(....)
and let the compiler do the hard work!